integration/tests/main/07-upgrade.spec.js

```diff
@@ -33,9 +33,13 @@ test("Upgrades an application", async ({ page }) => {
   // Deploy package
   await page.click('cds-button:has-text("Deploy") >> nth=0');
 
-  // Set replicas
+  // Increase replicas
   await page.locator("input[type='number']").fill("2");
-  await page.click('li:has-text("Changes")');
+
+  // Wait until changes are applied (due to the debounce in the input)
+  await page.waitForTimeout(1000);
```

Comment:
Using `waitForTimeout` is always a potential source of problems.
When having problems or debugging, it gives a bit more information to use  something like 

```
const locator = page.locator("input[type='number']")
await expect(locator).toHaveValue('2', {timeout: 1000})
```

BTW is there only one input of type number for this chart? Selector is very generic.

Reply:
Ok, so it is the event that we are waiting here.
Could we add more info in the comment to specify that it is the event please?
Not sure if [Playwright's `waitForEvent` + predicate func](https://playwright.dev/docs/api/class-page#page-wait-for-event) will help here.

Reply:
Thanks for the tip, I've been exploring it, but unfortunately, the events it listens to are not HTML events, but some playwright-specific ones: `backgroundpage`, `close`, `page`, `request`, `requestfailed`, `requestfinished`, `response`, `serviceworfker`

In SO, some responses point to more sophisticated alternatives (ideally for more denounced inputs). Example: https://stackoverflow.com/questions/73739729/how-to-test-expected-delay-time-on-page-element-fixed-wait-debounce-with-play

So, for now, I think we should keep it simple, and leave the wait :(